### DISCUSSION TITLE
Generating chromosomes

### DISCUSSION THREAD

**Discussion by BisUndefined** [PasaOpasen/geneticalgorithm2-6.8.7#59] [Show and tell]
First of all, thank you so much for this amazing work ! It helped me a lot !

I am currently working on a project where I create my chromosomes following a binary tree. So there are several conditions for these chromosomes to be viable. However, I can't find a way to make sure that the population is generated according to a specific function.
Is there a way to overcome this obstacle?

**Comment by PasaOpasen** [comment]
I don't understand which problem u have. What exactly (algorithmically) u want to do?
